backend/Object.h

```diff
@@ -13,6 +16,8 @@ class Object {
 		Object ();
 		
 		Object (Point point);
+
+		virtual void makeVirtual () {}
```

Comment:
This seems a bit.. hacky, but I don't know :woman_shrugging:

Reply:
I think I could make `pointsToString` virtual and rename it to `toString`, then I can add it to `Circle` and add the radius in it. So it would have a purpose rather than just being there. 🐤

Reply:
That sounds great :hatched_chick: